src/cache/precomputed_quantities.jl

```diff
@@ -488,33 +557,38 @@ function set_precomputed_quantities!(Y, p, t)
             ᶜρʲ = ᶜρʲs.:($j)
             ᶜq_totʲ = ᶜq_totʲs.:($j)
 
-            # boundary condition
             ρaʲ_level = Fields.field_values(Fields.level(ᶜρaʲ, 1))
             u³ʲ_halflevel = Fields.field_values(Fields.level(ᶠu³ʲ, half))
             Kʲ_level = Fields.field_values(Fields.level(ᶜKʲ, 1))
             h_totʲ_level = Fields.field_values(Fields.level(ᶜh_totʲ, 1))
             q_totʲ_level = Fields.field_values(Fields.level(ᶜq_totʲ, 1))
             tsʲ_level = Fields.field_values(Fields.level(ᶜtsʲ, 1))
             ρʲ_level = Fields.field_values(Fields.level(ᶜρʲ, 1))
-            ρ_level = Fields.field_values(Fields.level(Y.c.ρ, 1))
-            uₕ_level = Fields.field_values(Fields.level(Y.c.uₕ, 1))
-            h_tot_level = Fields.field_values(Fields.level(ᶜh_tot, 1))
-            q_tot_level = Fields.field_values(Fields.level(q_tot, 1))
-
-            p_level = Fields.field_values(Fields.level(ᶜp, 1))
-            Φ_level = Fields.field_values(Fields.level(ᶜΦ, 1))
 
-            local_geometry_level = Fields.field_values(
-                Fields.level(Fields.local_geometry_field(Y.c), 1),
+            @. u³ʲ_halflevel = CT3(
+                Geometry.WVector(FT(0.01), local_geometry_halflevel),
```

Comment:
We need something non zero here, but the value doesn't seem to be very important, so I'm hard coding it without using a parameter for now.